src/components/Kv/KvTooltip.vue

```diff
@@ -0,0 +1,274 @@
+		}
+	},
+	beforeDestroy() {
+		this.removeEvents();
+		if (this.popper) {
+			this.popper.destroy();
+		}
+	},
+	methods: {
+		open() {
+			this.setTimeout(() => {
+				this.show = true;
+				if (this.usingTouch) {
+					this.attachBodyEvents();
+				}
+			}, this.openDelay);
+		},
+		close() {
+			this.setTimeout(() => {
+				this.show = false;
+				if (this.usingTouch) {
+					this.removeBodyEvents();
+				}
+			}, this.closeDelay);
+		},
+		toggle() {
+			if (this.show) {
+				this.close();
+			} else {
+				this.open();
+			}
+		},
+		initPopper() {
+			this.popper = new Popper(this.reference, this.$el, {
+				placement: 'top',
+				modifiers: {
+					applyStyle: data => {
+						this.styles = data.styles;
+						this.setAttributes(data.attributes);
+					},
+					preventOverflow: {
+						padding: 10,
+					}
+				}
+			});
+		},
+		bodyTouchHandler(e) {
+			if (!isTargetElement(e, [this.reference, this.$el])) {
+				this.show = false;
+				this.removeBodyEvents();
+			}
+		},
+		referenceTapHandler(e) {
+			e.preventDefault();
+			this.toggle();
+		},
+		attachEvents() {
+			this.reference.addEventListener('mouseover', this.open);
+			this.reference.addEventListener('mouseout', this.close);
+			this.$el.addEventListener('mouseover', this.open);
+			this.$el.addEventListener('mouseout', this.close);
+			this.reference.addEventListener('touchstart', this.referenceTapHandler);
+		},
+		attachBodyEvents() {
+			onBodyTouchstart(this.bodyTouchHandler);
+		},
+		removeEvents() {
+			this.removeMouseEvents();
+			this.removeBodyEvents();
+			this.reference.removeEventListener('touchstart', this.referenceTapHandler);
+		},
+		removeMouseEvents() {
+			this.reference.removeEventListener('mouseover', this.open);
+			this.reference.removeEventListener('mouseover', this.close);
+			this.$el.removeEventListener('mouseover', this.open);
+			this.$el.removeEventListener('mouseout', this.close);
+		},
+		removeBodyEvents() {
+			offBodyTouchstart(this.bodyTouchHandler);
+		},
+		setAttributes(attrs) {
+			_map(attrs, (value, attr) => {
+				if (value === false) {
+					this.$el.removeAttribute(attr);
+				} else {
+					this.$el.setAttribute(attr, value);
+				}
+			});
+		},
+		setTimeout(fn, delay) {
+			window.clearTimeout(this.timeout);
+			this.timeout = window.setTimeout(fn, delay);
+		}
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.fade-enter-active,
+.fade-leave-active {
+	transition: opacity 0.3s;
+}
+
+.fade-enter,
+.fade-leave-to {
+	opacity: 0;
+}
+
+.tooltip-pane {
+	position: absolute;
+	background: $aqua-light-green;
+	border-radius: rem-calc(3);
+	box-shadow: 0 2px 0 rgba(196, 231, 219, 0.8);
+	border: 1px solid $aqua-medium-green;
+}
+
+.tooltip-content {
+	padding: 1rem;
+	max-width: rem-calc(250);
+	line-height: 1.4;
+}
+
+.tooltip-content .title-slot {
+	font-weight: 800;
+	margin-bottom: 0.75rem;
+}
+
+.tooltip-arrow {
+	width: 0;
+	height: 0;
+	border-style: solid;
+	position: absolute;
+	margin: 14px;
```

Comment:
The styles for the Caret/Arrow are from the popper site. Suppose I could set a var for rem-calc(14) and reuse it everywhere...